setup.py

```diff
@@ -0,0 +1,42 @@
+"""Setup for kinescope XBlock."""
+
+
+import os
+
+from setuptools import setup
+
+
+def package_data(pkg, roots):
+    """Generic function to find package_data.
+
+    All of the files under each of the `roots` will be declared as package
+    data for package `pkg`.
+
+    """
+    data = []
+    for root in roots:
+        for dirname, _, files in os.walk(os.path.join(pkg, root)):
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
+
+    return {pkg: data}
+
+
+setup(
+    name='kinescope-xblock',
+    version='0.1',
```

Comment:
```suggestion
    version='0.1.0',
```

Let's use semver.

If we don't envision further development before pushing this to a production instance, we should use:
```suggestion
    version='1.0.0',
```

Reply:
Based on the outcome of the discussion above, we might change this to cookiecutter template pretty soon.